src/main/java/ru/practicum/shareit/item/repository/ItemRepositoryInMemory.java

```diff
@@ -0,0 +1,77 @@
+package ru.practicum.shareit.item.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+import ru.practicum.shareit.item.model.Item;
+
+import java.util.List;
+import java.util.Map;
+
+@Repository
+@RequiredArgsConstructor
+public class ItemRepositoryInMemory implements ItemRepository {
+    private final Map<Long, Item> itemMap;
```

Comment:
Лучше здесь явно инициализировать мапу
```suggestion
    private final Map<Long, Item> itemMapc = new HashMap<>();
```